components/public-api-server/pkg/apiv1/identityprovider.go

```diff
@@ -79,6 +88,9 @@ func (srv *IdentityProviderService) GetIDToken(ctx context.Context, req *connect
 	userInfo := oidc.NewUserInfo()
 	userInfo.SetName(user.Name)
 	userInfo.SetSubject(subject)
+	if email != "" {
```

Comment:
You could drop the `if`, it would set it to the empty string, which is the default value in the struct anyway - it's omitempty in the JSON struct

Reply:
Indeed. I like the explicitness of this code though :)